src/services

```diff
@@ -725,6 +725,22 @@ p({
   "client":"{\"of\":\"foam.nanos.pm.PM\", \"remoteListenerSupport\": false}"
 })
 
+p({
+  "class":"foam.nanos.boot.NSpec",
```

Comment:
Is this line needed?

Reply:
Seems to be - it disappears from services if I remove this line